stylesheets/scss/_layout.scss

```diff
@@ -2,3 +2,45 @@
 	Layout goes in this file. Susy setup should go here too.
 	Layout classes should be prefixed `.l-`
 */
+
+// Main Configuration options for the Susy Grid
+$gutter-width: 20px;
+$column-count: 12;
+
+$susy: (
+  columns: $column-count,
+  use-custom: (rem:true),
+  global-box-sizing: border-box,
+  gutter-position: inside-static,
+  column-width: 96px,
+  gutters: $gutter-width/96px,
+  // debug: (
+  //   image: show-columns,
+  //   output: overlay,
+  //   toggle: bottom left,
+  // )
+);
+
+%l-container,
+.l-container {
+  //@include container($breakpoint-largest);
+  @include clearfix;
```

Comment:
@nathanshubert  It would come from here: 
https://github.com/domain7/sassyplate/blob/master/stylesheets/scss/_mixins.scss

Which I think we may want to update that one too